tests/prometheus/basic.py

```diff
@@ -0,0 +1,194 @@
+import asyncio
+from typing import Any, Optional, Type
+from unittest.mock import ANY, Mock, call
+
+import pytest
+from prometheus_client import CollectorRegistry
+
+from faststream.broker.core.usecase import BrokerUsecase
+from faststream.broker.message import AckStatus, StreamMessage
+from faststream.exceptions import RejectMessage
+from faststream.prometheus.middleware import (
+    PROCESSING_STATUS_BY_ACK_STATUS,
+    PROCESSING_STATUS_BY_HANDLER_EXCEPTION_MAP,
+    BasePrometheusMiddleware,
+)
+from faststream.prometheus.types import ProcessingStatus
+from tests.brokers.base.basic import BaseTestcaseConfig
+
+
+@pytest.mark.asyncio
+class LocalPrometheusTestcase(BaseTestcaseConfig):
+    broker_class: Type[BrokerUsecase]
```

Comment:
Please, do not use class-variables, use `self.get_broker` instead

Reply:
fixed